connect-location/src/main/java/com/ifttt/location/ConnectLocation.java

```diff
@@ -85,7 +87,10 @@ public static synchronized ConnectLocation init(
     }
 
     public static ConnectLocation getInstance() {
-        checkInit(INSTANCE);
+        if (INSTANCE == null) {
+            throw new IllegalStateException(
+                "ConnectLocation is not initialized correctly. Please call ConnectLocation.init first.");
+        }
```

Comment:
`checkInit` is removed?

Reply:
Yeah, there is no need to do that if we only have one place where we need to check this.